scripts/add_image.sh

```diff
@@ -2,8 +2,10 @@ node dist/index.js addimage \
     -r "service-endpoint" \
     -u "user_address" \
     -x "private_key" \
-    --path "image.wasm" \
-    --name "Image name"
+    --path "image.wasm"
+
```

Comment:
Better add the --description in this example so user can add description as it is shown in our UI.